src/preact/slot.js

```diff
@@ -53,11 +56,11 @@ export function createSlot(element, name, defaultProps, as = false) {
 /**
  * Slot component.
  *
- * @param {!JsonObject} props
- * @return {!PreactDef.VNode}
+ * @param {JsonObject} props
+ * @return {VNode}
  */
 export function Slot(props) {
-  const ref = useRef(/** @type {?Element} */ (null));
+  const ref = useRef(/** @type {HTMLSlotElement|null} */ (null));
```

Comment:
I suspect the cast isn't necessary here at all

Reply:
Nvm; it probably infers the generic type to be `null`, based on your comment above

Reply:
You were right! Removing

Reply:
Odd